modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,196 @@
+
+| https://issues.couchbase.com/browse/MB-52000[MB-52000^]
+| *Summary*: Alter Index failure when nodes have un encrypted ports in encryption mode all
+
+|===
+
+=== Search Service
+
+[#table_fixedissues_v704-searchservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-50946[MB-50946^]
+| *Summary*: bleveMaxClauseCount reset to 1024 after restart
+
+|===
+
+=== Analytics Service
+
+[#table_fixedissues_v704-analyticsservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51676[MB-51676^]
+| *Summary*: To investigate the crash in MB-49471
+
+| https://issues.couchbase.com/browse/MB-51483[MB-51483^]
+| *Summary*: Data is not completely ingested into datasets
+
+| https://issues.couchbase.com/browse/MB-51482[MB-51482^]
+| *Summary*: Node is not upgraded and stuck trying to register with CC
+
+| https://issues.couchbase.com/browse/MB-51488[MB-51488^]
+| *Summary*: Ignore byte order marks when parsing external data
+
+| https://issues.couchbase.com/browse/MB-51700[MB-51700^]
+| *Summary*: CBAS service keeps crashing when one of the CBAS nodes is failed over
+
+| https://issues.couchbase.com/browse/MB-51412[MB-51412^]
+| *Summary*:  Graceful failover + delta recovery + rebalance fails
+
+|===
+
+=== Eventing Service
+
+[#table_fixedissues_v704-eventingservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51585[MB-51585^]
+| *Summary*: Limit ownership_history to last 'N' in metadata bucket
+
+| https://issues.couchbase.com/browse/MB-51496[MB-51496^]
+| *Summary*: Eventing functions lose auth on save
+
+| https://issues.couchbase.com/browse/MB-51497[MB-51497^]
+| *Summary*: Warning if variable name contains builtin api name
+
+| https://issues.couchbase.com/browse/MB-51231[MB-51231^]
+| *Summary*: Memory leak when performing undeploy and pause operation
+
+| https://issues.couchbase.com/browse/MB-51379[MB-51379^]
+| *Summary*: Rollback handling - Retry with previous uuid on rollback message
+
+| https://issues.couchbase.com/browse/MB-51688[MB-51688^]
+| *Summary*: Refresh connectionpool on every encryption-level change
+
+| https://issues.couchbase.com/browse/MB-51390[MB-51390^]
+| *Summary*: Handler stuck in pausing state after online upgrade and enforcing TLS
+
+| https://issues.couchbase.com/browse/MB-51687[MB-51687^]
+| *Summary*: Some bucket operations are failing with LCB_ERR_SSL_CANTVERIFY after online upgrade to 7.0.x
+
+| https://issues.couchbase.com/browse/MB-51386[MB-51386^]
+| *Summary*: Concurrent attempts to restart a crashed consumer process can cause rebalance & life cycle ops to fail or get stuck
+
+|===
+
+[#known-issues-704]
+== Known Issues
+
+=== Install and Upgrade
+
+[#table_fixedissues_v704-installandupgrade,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52313[MB-52313^]
+| *Summary*: Analytics Service crashes during upgrade
```

Comment:
We should mention this is only on Windows.  And there's a workaround per the MB comments.

Reply:
We have to follow up with the workaround, I understand. I can add that this is windows.